gpkit/nomials/nomial_math.py

```diff
@@ -103,10 +101,10 @@ def __init__(self, exps=None, cs=1, require_positive=True, simplify=True,
                 cs = cs.to(units)
 
         # init NomialData to create self.exps, self.cs, and so on
-        super(Signomial, self).__init__(exps, cs, simplify=simplify)
+        super(Nomial, self).__init__(exps, cs, simplify=simplify)
```

Comment:
did this need to change from `Signomial` to `Nomial`? usual convention is to call `super` witt the child class as the first argument. Unit tests run fine with it changed to `Signomial`.